storage_service/common/management/commands/import_aip.py

```diff
@@ -0,0 +1,405 @@
+``--aip-storage-location``) as well as the UUID of the pipeline that the AIP
+was created with (using ``--pipeline``).
+
+The command will:
+
+- validate the AIP (make sure it is a valid Bag),
+- decompress the AIP (if told to do so),
+- copy the AIP to a local filesystem AIP Storage location,
+- compress the AIP (if told to do so),
+- create a database row (in the locations_package table) for the AIP in the
+  Storage Service database, and
+- call the AIP model instance's ``store_aip`` method, thereby moving the AIP to
+  its ultimate destination space/location (as specified by the caller) and
+  creating a pointer file (if needed).
+
+Here is an example of running this command in an am.git Docker Compose deploy
+to import an uncompressed AIP new_uncompressed.tar.gz ::
+
+    $ make manage-ss ARG='import_aip /home/archivematica/new_uncompressed.tar.gz --decompress-source --compression-algorithm="7z with bzip"'
+
+Note that in the above command we are importing a .tar.gz file that---once
+uncompressed---is expected to be an uncompressed AIP. That's what the
+``--decompress-source`` flag means. The ``--compression-algorithm`` option
+indicates which compression algorithm to use for the final, compressed AIP. The
+final AIP in this case will be compressed and will have a pointer file.
+
+To get help::
+
+    $ make manage-ss ARG='import_aip -h'
+
+Question: Why are the permissions 664 on an imported AIP but 775 on one created
+via a pipeline? Should this be fixed by the import command?::
+
+    $ ls -lh /tmp/am-pipeline-data/www/AIPsStore/0fde/dcc1/599d/4f6e/b132/4b7b/c035/5b1f/CVA1426-0fdedcc1-599d-4f6e-b132-4b7bc0355b1f.7z
+    -rw-rw-r--  1 joeldunham  wheel    25M 30 Jul 23:03 /tmp/am-pipeline-data/www/AIPsStore/0fde/dcc1/599d/4f6e/b132/4b7b/c035/5b1f/CVA1426-0fdedcc1-599d-4f6e-b132-4b7bc0355b1f.7z
+    $ ls -lh /tmp/am-pipeline-data/www/AIPsStore/237e/12b4/03c9/451a/9be7/4915/b0bd/9012/FakeCVA2-237e12b4-03c9-451a-9be7-4915b0bd9012.7z
+    -rwxrwxr-x  1 joeldunham  wheel    51M 30 Jul 22:58 /tmp/am-pipeline-data/www/AIPsStore/237e/12b4/03c9/451a/9be7/4915/b0bd/9012/FakeCVA2-237e12b4-03c9-451a-9be7-4915b0bd9012.7z
+
+"""
+
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import glob
+import logging
+import os
+from pwd import getpwnam
+import shlex
+import shutil
+import subprocess
+import tarfile
+import tempfile
+
+import bagit
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
+
+from administration.models import Settings
+from common import utils
+from locations import models
+
+
+# Suppress the logging from models/package.py
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': True,
+})
+
+
+DEFAULT_AS_LOCATION = 'default_AS_location'
+DEFAULT_UNIX_OWNER = 'archivematica'
+ANSI_HEADER = '\033[95m'
+ANSI_OKGREEN = '\033[92m'
+ANSI_WARNING = '\033[93m'
+ANSI_FAIL = '\033[91m'
+ANSI_ENDC = '\033[0m'
```

Comment:
You may want to revisit some of the stuff you're doing around logging. It'd be good to take a look at https://github.com/django/django/tree/stable/1.8.x/django/core/management and see how things are done in Django, e.g.: see `colors`, `DJANGO_COLORS`, exceptions like `CommandError`, etc...